sys/net/gnrc/netif/gnrc_netif_device_type.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @brief   Functions that are based around a network interface's device type.
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#include "log.h"
+#include "net/gnrc/netif.h"
+#include "net/ethernet.h"
+#include "net/ieee802154.h"
+
+#ifdef MODULE_GNRC_IPV6
+#if defined(MODULE_CC110X) || defined(MODULE_NRFMIN)
+static void _create_iid_from_short(const uint8_t *addr, size_t addr_len,
+                                   eui64_t *iid)
+{
+    const unsigned offset = sizeof(eui64_t) - addr_len;
+
+    memset(iid->uint8, 0, sizeof(iid->uint8));
+    iid->uint8[3] = 0xff;
+    iid->uint8[4] = 0xfe;
+    memcpy(&iid->uint8[offset], addr, addr_len);
+}
+#endif /* defined(MODULE_NETDEV_IEEE802154) || defined(MODULE_CC110X) ||
```

Comment:
comment does not match the if block above

Reply:
Fixed